.github/workflows/ci.yml

```diff
@@ -6,9 +6,9 @@ on:
       - master
   pull_request:
   schedule:
-    # Schedule monthly checks on master, to ensure that CI does not silently break
-    # when nothing is pushed for a while.
-    - cron: '30 3 1 * *'
+    # Schedule weekly builds on master, to ensure that the Docker image gets
+    # rebuilt regularly (in order to include security fixes in the base image).
+    - cron: '5 4 * * 5'
```

Comment:
Of course a prerequisite is that the base image is also rebuilt regularly. We should probably add a weekly CI step there as well (to be built a few hours before this image is built).

Reply:
Yes, that would be great for automated container updates (in case security things are fixed in the upstream base image).

In GitLab CI, I would include the base image build also in this repository and add a conditional build job if the base image Dockerfile changes. However, this is currently not possible with GitHub Actions (I wrote a feature request for that some time ago).

I opened an issue for that https://github.com/studentenportal/docker-studentenportal-base/issues/2.